spec/jobs/submission_ingestion_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe SubmissionIngestionJob do
+  describe '#perform' do
+    context 'when a submission is marked as ingest_failed' do
+      let(:submission_file) { create(:submission_file, submission: build(:submission, aasm_state: :ingest_failed)) }
+
+      it 'raises an exception that will discard the job' do
```

Comment:
It's occurred to me after the fact that it might be nice to have an inverse test here, to make sure that other exceptions still cause a retry, but I don't think it's really necessary.